code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -498,7 +498,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 /datum/reagent/consumable/ethanol/brave_bull/on_mob_delete(mob/living/M)
 	to_chat(M, "<span class='notice'>You no longer feel [tough_text].</span>")
 	M.maxHealth -= 10
-	M.health -= 10 //This can indeed crit you if you're alive solely based on alchol ingestion
+	M.health = min(M.health - 10, M.maxHealth) //This can indeed crit you if you're alive solely based on alchol ingestion
```

Comment:
I'm fucking wrong
```
if(M.health > M.maxHealth)
    M.health = M.maxHealth
```

Reply:
Joy okay hand one hundred

Reply:
This should still do that, shouldn't it? (I used `min()` and not `max()`)

Reply:
The current version can still crit people

Reply:
That's intentional, unless you'd rather I remove that?

Reply:
Cough

Reply:
"Would you have this removed"
4 DAYS OF SILENCE PRIORITY MOVED TO OTHER PRs AND WILL MARK AS STALE WHEN NOTICED AGAIN.